templates/feed.jinja

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="en">
-    <generator uri="{{ site.url }}" version="0.1.0">{{ site.name }}</generator>
-    <link href="{{ site.url }}/feed.xml" rel="self" type="application/atom+xml" />
-    <link href="{{ site.url }}" rel="alternate" type="text/html" />
-    <id>{{ site.url }}</id>
+<feed xmlns="http://www.w3.org/2005/Atom">
     <title>{{ site.name }}</title>
     <subtitle>{{ site.description }}</subtitle>
+    <updated>{{ now() }}</updated>
+    <id>{{ site.url }}</id>
+    <link href="{{ site.url }}/feed.xml" rel="self" type="application/atom+xml" />
+    <link href="{{ site.url }}" rel="alternate" type="text/html" />
+    <generator uri="https://github.com/zineland/zine" version="{{ generator_version }}">zine</generator>
```

Comment:
I think we can use `get_env()` function, see https://tera.netlify.app/docs/#get-env

Reply:
> I think we can use `get_env()` function, see https://tera.netlify.app/docs/#get-env

`CARGO_PKG_VERSION` seems to be available only when compiling. So we may set the version of the zine at the compile time. Otherwise, the template engine may panic:

```
Error: Failed to render 'feed.jinja'

Caused by:
    0: Function call 'get_env' failed
    1: Environment variable `CARGO_PKG_VERSION` not found
```